master/templates/admin/master/city/change_form.html

```diff
@@ -61,16 +61,16 @@
 	console.log(cityname);
 	alert(cityname);
 
-        var id = django.jQuery("#id_name option:selected").val();
-        alert(id);
+  var id = django.jQuery("#id_name option:selected").val();
+  alert(id);
         
 	
 
 django.jQuery.ajax({
  
-    url: "http://127.0.0.1:8000/admin/search/",
+    url : "http://127.0.0.1:8000/admin/search/",
```

Comment:
Remove this hardcoded URL and use {% url 'some-url-name' arg1=v1 arg2=v2 %}  this returns an absolute path reference matching the given view.